shell/spy_test.go

```diff
@@ -1,14 +1,38 @@
 package shell
 
 import (
-	"fmt"
 	"testing"
+
+	"github.com/google/go-cmp/cmp"
 )
 
-func TestExecute(t *testing.T) {
-	want := fmt.Sprintf("spy.go\nspy_test.go\n")
-	got := Execute("ls")
-	if got != want {
-		t.Errorf("Expected: %s, got: %s", want, got)
+func TestParseCommand(t *testing.T) {
+	tcs := []struct {
+		cmd  string
+		name string
+		args []string
+	}{
+		{"echo Bukola", "echo", []string{"Bukola"}},
+		{"test it works fine", "test", []string{"it", "works", "fine"}},
```

Comment:
We should add a test case with unbalanced quotes, which forces us to check the `ok` value received from `shell.Split`, shouldn't we? What should we do in that case? Return error from `ParseCommand`? Engineering decision needed 😁